src/Iban.php

```diff
@@ -2,14 +2,13 @@
 
 namespace IbanGenerator;
 
-use IbanGenerator\Bban\BbanInterface;
-use IbanGenerator\Bban\SpainBban;
+use IbanGenerator\Bban;
```

Comment:
Looks like this is no used anymore, maybe you can use Bban\BbanInterface; instead